modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,166 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-using-assisted-installer-oci_{context}"]
+= Using the Assisted Installer to install a cluster on OCI
+
+In {product-title} {product-version}, you can use the Assisted Installer to install a cluster on {oci-first} by using infrastructure that you provide.
+
+:FeatureName: Using the Assisted Installer to install an {product-title} cluster on OCI
+include::snippets/technology-preview.adoc[]
+
+You can access {oci} Resource Manager configurations to provision and configure {oci} resources that meet your organization's needs.
+
+[IMPORTANT]
+====
+The steps for provisioning {oci} resources are provided as an example only. You can also choose to create the required resources through other methods; the scripts are just an example. Installing a cluster with infrastructure that you provide requires knowledge of the cloud provider and the installation process on {product-title}. You can access {oci} Resource Manager configurations to complete these steps, or use the configurations to model your own custom script.
+====
+
+Follow the steps in the procedure to understand how to use the Assisted Installer to install a {product-title} cluster on OCI. The procedure demonstrates the use of the {oci} Manager (CCM) and Oracle’s Container Storage Interface (CSI) objects to link your {product-title} cluster with the {oci} API.
+
+.Prerequisites
+
+* You reviewed details about the {product-title} xref:../../architecture/architecture-installation.adoc#architecture-installation[installation and update] processes. 
+//* You completed the [Request Access to Red Hat OpenShift on Oracle Cloud Infrastructure in Developer Preview](https://docs.google.com/forms/d/e/1FAIpQLSdQHmw0BUWjQxPKC5_G7lk2_o9Pcnn2ON84Al6s2Ir254gv6Q/viewform) form.
```

Comment:
SME INPUT REQUIRED: Is this form still a requirement? The feature is now tech preview.

Reply:
@faermanj